projects/common/src/time/time-range.service.test.ts

```diff
@@ -119,16 +120,19 @@ describe('Time range(TR) service', () => {
             addQueryParametersToUrl: (newParams: QueryParamObject) => mockNavigation$.next(newParams.time as string),
             getQueryParameter: jest
               .fn()
-              .mockReturnValueOnce('1573277100277-1573277100277')
-              .mockReturnValueOnce('1573255100253-1573255111159')
-              .mockReturnValue('1573255111159-1573455111990'),
+              .mockReturnValueOnce(invalidTimeRange.toUrlString())
+              .mockReturnValueOnce(defaultTimeRange.toUrlString())
```

Comment:
this one we wouldn't want to come from the query param, it's the default time range (not sure why it did before). Similarly line 134 shouldn't be set as the default. It would probably be more straight forward to split this into a different test, see that the default gets set when the url returns invalid.